third_party/2and3/dateutil/tz/__init__.pyi

```diff
@@ -10,3 +10,5 @@ from .tz import (
     datetime_exists as datetime_exists,
     datetime_ambiguous as datetime_ambiguous,
 )
+
+UTC = ...  # type: tzutc
```

Comment:
Please use variable annotations for new and changed code:

```suggestion
UTC: tzutc
```

Reply:
Oh sorry, I thought it's related to being 2and3 or something. :)